features/siddhi-tryit/org.wso2.carbon.siddhi.tryit.feature/pom.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@keizer619 
Fixed in commit https://github.com/wso2/carbon-event-processing/pull/179